docs/src/environments/search-engine-visibility.md

```diff
@@ -22,34 +24,37 @@ title=In the Console
 +++
 
 - Select the project where you want to change visibility.
-- From the **Environment** menu, select the environment you want to change.
+- From the **Environment** menu, select the Production environment you want to change.
 - Click {{< icon settings >}} **Settings**.
 - In the row with **Hide from search engines**, click **Edit {{< icon chevron >}}**.
 - Select or clear the **Hide from search engines** checkbox.
 
 <--->
+
 +++
 title=Using the CLI
 +++
 
 Run the following command:
 
 ```bash
-platform environment:info -e <ENVIRONMENT_NAME> restrict_robots true
+platform environment:info --environment {{< variable "PRODUCTION_ENVIRONMENT_NAME" >}} restrict_robots true
 ```
 
 {{< /codetabs >}}
 
 ## How it's done
 
-Search engines are turned away from environments by including a `X-Robots-Tag` header:
+When the **Hide from search engines** is activated,
+search engines are turned away from environments by including a `X-Robots-Tag` header:
 
 ```txt
 X-Robots-Tag: noindex, nofollow
 ```
 
 That tells search engine indexers to not index these sites and not traverse links from these sites.
-This helps keep non-Production sites out of search engine indexes.
+This keeps non-Production sites out of search engine indexes.
+For non-Production sites it can't be turned off.
```

Comment:
:hammer_and_wrench: This isn't the place for this information. Plus it doesn't work absolutely.
```suggestion
This helps keep non-Production sites out of search engine indexes.
```

Reply:
Why doesn't it work absolutely? Where would be the place for the information that it can't be turned off for non-prod?